src/performance_counters/register_counters.cpp

```diff
@@ -32,6 +32,9 @@ namespace phylanx { namespace performance_counters
           : hpx::performance_counters::base_performance_counter<
                 primitive_counter>(info)
         {
+            hpx::performance_counters::counter_path_elements paths;
+            hpx::performance_counters::get_counter_path_elements(info.fullname_, paths);
+            duration_counter = paths.countername_.find("count") != std::string::npos;
```

Comment:
`duration_counter` will be true for performance counters that are supposed to report count numbers, not durations.

Reply:
Fixed.